packages/core/src/components/forms/_form-group.scss

```diff
@@ -0,0 +1,144 @@
+// Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+// of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+// and https://github.com/palantir/blueprint/blob/master/PATENTS
+
+@import "../../common/variables";
+@import "../../common/mixins";
+
+/*
+Form groups
+
+Form groups allow for more complex forms and fields than [`Labels`](#components.forms.label) allow
+(eg. when used with [`Numeric inputs`](#components.forms.numeric-input). They also allow the addition of helper text to aid with
+user navigation.
+
+<div class="pt-callout pt-intent-primary pt-icon-info-sign">
+  Note that this feature necessitates additional DOM elements, thus requiring a more complex structure achieved with `.pt-form-group`.
+</div>
+
+- Link each label to its respective control element with a `for={#id}` attribute on the `<label>` and
+`id={#id}` on the control.
+
+- Add `.pt-intent-*`, `.pt-disabled`  to `.pt-form-group` to style the label and helper text (similar to labels, nested controls need to be styled separately).
+
+- Add `.pt-inline` to `.pt-form-group` to place the label to the left of the control.
+
+- Add `.pt-large` to `.pt-form-group` to align the label when used with large inlined Blueprint controls.
+
+Markup:
+<div class="pt-form-group">
+  <label class="pt-label" for="example-form-group-input-a">
+    Label A
+    <span class="pt-label-description">(required)</span>
+  </label>
+  <div class="pt-form-content">
+    <div class="pt-input-group">
+      <span class="pt-icon pt-icon-calendar"></span>
```

Comment:
I'd put the icon in the next example so we can see how it renders with an intent